products/sle/main.pm

```diff
@@ -1009,7 +990,6 @@ else {
             loadtest "ses/deepsea_testsuite";
```

Comment:
In the code 2 lines above `boot_to_desktop` is called, maybe it can be replaced by `boot_hdd_image`?

Who can have a look? @dzedro maybe?

Reply:
it can be replaced, with support server it will wait for network setup on support server

Reply:
It will not work, boot_to_desktop is doing also login
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/boot/boot_to_desktop.pm

Reply:
`boot_hdd_image` call `boot_to_desktop`:https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/b6d7d10096eddbceb120c91c49b1d25f0999366e/lib/main_common.pm#L665.
So it should be fine.